osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/infra/messaging/processors/GetOutagesRequestMessageProcessor.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
```

Comment:
This file looks new, copyright should be for Alliander N.V. instead of Smart Society Services B.V.